packages/fxa-auth-server/lib/routes/subscriptions/mozilla.ts

```diff
@@ -92,12 +92,14 @@ export class MozillaSubscriptionHandler {
 
     const targetPlanId = request.params.planId;
 
-    const eligibility = (
-      await this.capabilityService.getPlanEligibility(uid, targetPlanId)
-    )[0];
+    const eligibility = await this.capabilityService.getPlanEligibility(
+      uid,
+      targetPlanId
+    );
 
     return {
-      eligibility,
+      eligibility: eligibility[0],
+      currentPlan: eligibility[1],
```

Comment:
[question] Should this be called `currentPlan` or `blockingPlan`?

Reply:
I would keep this as `currentPlan` since in all cases (e.g., upgrade, downgrade, or IAP roadblock), it represents the current plan that the customer is subscribed to before their attempts to change to a new plan. It's also not always a blocking plan if the upgrade is successful.

Reply:
Sounds good!